### PR TITLE
Adding more priority labels

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
The intent is to help a queue of pull requests (PRs) with approriate
priorities, so that we will have a clear queue of PRs which needs review

Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
label priority/unspecified

File: README.md [hunk 1]
```diff
@@ -145,9 +145,13 @@ things that authors can do to help keep the conversation moving along.
...
 
-- `priority/important-soon` indicates that the enhancement is related to a
-top level release priority. These will be highlighted in the
-[this-week](this-week/) newsletters.
+- `priority/high`  indicates that the enhancement is related to a
+top level release priority. These will be highlighted in the [this-week](this-week/)
+newsletters.
+- `priority/medium`  indicates that the enhancement is related to a
+future release but it is not the next OCP release 
+- `priority/low` indicates that the enhancement is does not have a clear release date.
```
Comment (kikisdeliveryservice) [inline README.md:153]: I think priorit/low conflict with intention of closing PRs after a short period of time. perhaps same with medium.

**Review by kikisdeliveryservice — COMMENTED** [review]
I think we should add priority/critical-urgent
and remove priority medium & low.

File: README.md [hunk 2]
```diff
@@ -145,9 +145,13 @@ things that authors can do to help keep the conversation moving along.
...
 The following labels may be applied to enhancements to help categorize them:
 
-- `priority/important-soon` indicates that the enhancement is related to a
-top level release priority. These will be highlighted in the
-[this-week](this-week/) newsletters.
+- `priority/high`  indicates that the enhancement is related to a
+top level release priority. These will be highlighted in the [this-week](this-week/)
+newsletters.
+- `priority/medium`  indicates that the enhancement is related to a
+future release but it is not the next OCP release
```
Comment (jsafrane) [inline README.md:152]: There is a gap between high and medium - what priority has a feature that targets the next release, but is not as a "top level release priority"/

File: README.md (hunk 1 above)
Comment (kikisdeliveryservice) [reply README.md:153]: I think that makes sense

File: README.md (hunk 2 above)
Comment (LalatenduMohanty) [reply README.md:152]: yes, I have added `important-soon` and changed the wording for high priority as the discussion.

File: README.md (hunk 1 above)
Comment (LalatenduMohanty) [reply README.md]: Fixed it in https://github.com/openshift/enhancements/commit/fda887bedfd9cca56720a65d10fc07788be1b766. Take a look.

File: README.md [hunk 3]
```diff
@@ -144,10 +144,19 @@ things that authors can do to help keep the conversation moving along.
...
-[this-week](this-week/) newsletters.
+- `priority/important-soon` indicates that the enhancement pull request is related to a
+top level release priority. These will be highlighted in the [this-week](this-week/)
+- `priority/high`  indicates that relative priority of the enhancement as compared
+ to the other enhancement pull requests
+- `priority/medium`  indicates that relative priority of the enhancement as compared
+ to the other enhancement pull requests
+- `priority/low` indicates that relative priority of the enhancement as compared
+ to the other enhancement pull requests
+- `priority/unspecified`  indicates that the enhancement needs to go through triage
```
Comment (dhellmann) [inline README.md:155]: What is going to set `priority/unspecified`? Who is going to do the triage?

File: README.md [hunk 4]
```diff
@@ -144,10 +144,19 @@ things that authors can do to help keep the conversation moving along.
...
+
+*Note*: The priority of the enhancment will help creating the queue of PRs that needs
+ to get merged for OCP releases. It will also help to set expectation around how soon
+ the enhancment needs to get merged.
```
Comment (dhellmann) [inline README.md:159]:
```suggestion
 the enhancment needs to get reviewed.
```

We want to encourage review and discussion, but we don't want to imply that we set deadlines on merging designs.

File: README.md [hunk 5]
```diff
@@ -144,10 +144,19 @@ things that authors can do to help keep the conversation moving along.
...
 
 The following labels may be applied to enhancements to help categorize them:
-
-- `priority/important-soon` indicates that the enhancement is related to a
-top level release priority. These will be highlighted in the
-[this-week](this-week/) newsletters.
+- `priority/important-soon` indicates that the enhancement pull request is related to a
+top level release priority. These will be highlighted in the [this-week](this-week/)
+- `priority/high`  indicates that relative priority of the enhancement as compared
+ to the other enhancement pull requests
```
Comment (wking) [inline README.md:150]: [This comment][1] suggested that individual teams would be setting semantics, so do we need "to the other enhancement pull requests belonging to the relevant team"?  Or something that points out that this is a relative position in a per-team (or per-approver?) queue, and not a relative position in a single repo-wide queue?

[1]: https://github.com/openshift/enhancements/pull/1129#discussion_r879429987

**Comment by dhellmann** [comment]
@LalatenduMohanty are you still working on this?

**Pull request closed without merging**